src/main/steps/connection-summary/content.ts

```diff
@@ -0,0 +1,43 @@
+import { Checkbox } from '../../app/case/case';
+import { YesOrNo } from '../../app/case/definition';
+import { TranslationFn } from '../../app/controller/GetController';
+import { FormContent } from '../../app/form/Form';
+import { CommonContent } from '../common/common.content';
+import { habitualAndDomicileHelp } from '../you-can-use-english-welsh-courts/content';
+
+const en = ({ isDivorce, applyForDivorce, applyForDissolution, partner, formState }: CommonContent) => {
+  const apply = isDivorce ? applyForDivorce : applyForDissolution;
+  let reasonsToApplyInEnglandOrWales;
+  if (formState?.jurisdictionResidualEligible === YesOrNo.YES) {
+    reasonsToApplyInEnglandOrWales = 'the courts of England and Wales have jurisdiction on a residual basis';
+  } else if (formState?.lastHabituallyResident === YesOrNo.NO && formState?.sameSex === Checkbox.Checked) {
+    reasonsToApplyInEnglandOrWales = `your ${partner} is domiciled in England or Wales`;
+  } else {
+    reasonsToApplyInEnglandOrWales = 'you are domiciled in England and Wales';
```

Comment:
Could this logic be moved into `src/main/app/jurisdiction/connections.ts` or the existing logic be reused? So that we have one source of truth for how their connections to England/Wales are matched.

This way we can also then have an array of multiple connections for a multiple bullet list on the `/you-can-use-english-welsh-courts`

Reply:
after decision, it seems the page is different even if some content is similar and later on will have radio components.